### PR TITLE
Ingest as Makefile

### PR DESCRIPTION
**Pull request by daisieh**
Running `make all` should add the mcode-synthetic dataset to your docker-installed CanDIGv2 stack.

Note: if you've never installed htslib, you'll get an error message telling you to do that. Follow those instructions!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!make
+
+# import global variables
+CANDIG_HOME ?= ../CanDIGv2
+env ?= $(CANDIG_HOME)/.env
```
Comment (shaikh-rashid) [inline Makefile:5]: This is not recommended. What if my CanDIGv2 path is not in the same relative path as this repo?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ifeq (, $(shell which bgzip))
+$(error "bgzip is part of htslib; htslib is required to manage variant files: installation instructions are at https://www.htslib.org/download/")
+endif
+$(foreach F, $(wildcard samples/*.vcf), $(shell bgzip -i $(F)))
+
+/samples:
+	@mkdir -p $(DIR)/samples
+
+clinical_ETL.ready:
+	git clone https://github.com/CanDIG/clinical_ETL.git
```
Comment (shaikh-rashid) [inline Makefile:40]: This should be a submodule. You can then initialize submodules within Makefile

File: Makefile (hunk 1 above)
Comment (daisieh) [reply Makefile:5]: It's one of those "set if not already set" variables, so I could add a note at the top of the file to say that you should set the env var if that's not where your value is.

**Review by shaikh-rashid — APPROVED** [review]
LGTM

**Pull request merged**